java/src/main/java/com/start/Connection.java

```diff
@@ -1,19 +1,30 @@
 package com.start;
 
-import com.mongodb.client.MongoClient;
-import com.mongodb.client.MongoClients;
 import org.bson.Document;
 
-import java.util.ArrayList;
-import java.util.List;
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
 
 public class Connection {
 
     public static void main(String[] args) {
-        String connectionString = "mongodb://localhost:27017";
-        try (MongoClient mongoClient = MongoClients.create(connectionString)) {
-            List<Document> databases = mongoClient.listDatabases().into(new ArrayList<>());
-            databases.forEach(db -> System.out.println(db.toJson()));
+        for (String connectionUri : args) {
```

Comment:
I think it would be easier to accept just one URI and then just call this test twice in the dance configuration